src/components/VAudioTrack/VPlayPause.vue

```diff
@@ -41,13 +41,6 @@ import playIcon from '~/assets/icons/play.svg'
 import pauseIcon from '~/assets/icons/pause.svg'
 import replayIcon from '~/assets/icons/replay.svg'
 
-const statusVerbMap = {
-  playing: 'pause',
-  paused: 'play',
-  played: 'replay',
-  loading: 'loading',
-} as const
```

Comment:
Had to move this so that it could be accessible from Playwright as we don't have Playwright configured to know how to import from `.vue` SFCs.